esignet-default.properties

```diff
@@ -459,3 +461,13 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}
 
 logging.level.org.springframework.web.client.RestTemplate=DEBUG
 logging.level.io.mosip.esignet=DEBUG
+
+##---------------------------------sunbird-RC configuration------------------------------------------------------
```

Comment:
Rename this to Sunbird-RC Plugin Configurations

Reply:
update the same in other file also

Reply:
changed the name